### ISSUE TITLE
Unspecified error [659dcb50ed]

### ISSUE THREAD

**Issue by aem-intellij-plugin-error-reporter** [aem-intellij-plugin-error-reporter/auto-generated-issues-storage#59]
fixed:0.11.2

## Description
*No description*
## Exception
```
java.lang.ClassCastException: class com.intellij.openapi.file.exclude.EnforcedPlainTextFileTypeFactory$2 cannot be cast to class com.intellij.openapi.fileTypes.LanguageFileType (com.intellij.openapi.file.exclude.EnforcedPlainTextFileTypeFactory$2 and com.intellij.openapi.fileTypes.LanguageFileType are in unnamed module of loader com.intellij.util.lang.UrlClassLoader @56ef9176)
	at com.intellij.util.indexing.FileContentImpl.createFileFromText(FileContentImpl.java:138)
	at com.intellij.util.indexing.FileContentImpl.getFileFromText(FileContentImpl.java:106)
	at com.intellij.util.indexing.FileContentImpl.getPsiFileForPsiDependentIndex(FileContentImpl.java:274)
	at com.intellij.util.indexing.FileContentImpl.getPsiFile(FileContentImpl.java:94)
	at co.nums.intellij.aem.bC$b.a(Unknown Source)
	at co.nums.intellij.aem.bC$b.map(Unknown Source)
	at com.intellij.util.indexing.impl.MapReduceIndex.mapInput(MapReduceIndex.java:312)
	at com.intellij.util.indexing.VfsAwareMapReduceIndex.mapInput(VfsAwareMapReduceIndex.java:109)
	at com.intellij.util.indexing.impl.MapReduceIndex.calculateUpdateData(MapReduceIndex.java:268)
	at com.intellij.util.indexing.impl.MapReduceIndex.update(MapReduceIndex.java:250)
	at com.intellij.util.indexing.FileBasedIndexImpl.updateSingleIndex(FileBasedIndexImpl.java:1680)
	at com.intellij.util.indexing.FileBasedIndexImpl.lambda$doIndexFileContent$28(FileBasedIndexImpl.java:1605)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.freezeFileTypeTemporarilyIn(FileTypeManagerImpl.java:656)
	at com.intellij.util.indexing.FileBasedIndexImpl.doIndexFileContent(FileBasedIndexImpl.java:1564)
	at com.intellij.util.indexing.FileBasedIndexImpl.indexFileContent(FileBasedIndexImpl.java:1546)
	at com.intellij.util.indexing.FileBasedIndexImpl.processRefreshedFile(FileBasedIndexImpl.java:1528)
	at com.intellij.util.indexing.FileBasedIndexImpl$VirtualFileUpdateTask.doProcess(FileBasedIndexImpl.java:1707)
	at com.intellij.util.indexing.FileBasedIndexImpl$VirtualFileUpdateTask.doProcess(FileBasedIndexImpl.java:1704)
	at com.intellij.util.indexing.UpdateTask.process(UpdateTask.java:76)
	at com.intellij.util.indexing.UpdateTask.processAll(UpdateTask.java:46)
	at com.intellij.util.indexing.FileBasedIndexImpl.forceUpdate(FileBasedIndexImpl.java:1726)
	at com.intellij.util.indexing.FileBasedIndexImpl.ensureUpToDate(FileBasedIndexImpl.java:736)
	at com.intellij.util.indexing.FileBasedIndexImpl.ensureUpToDate(FileBasedIndexImpl.java:702)
	at com.intellij.psi.stubs.StubIndexImpl.doProcessStubs(StubIndexImpl.java:306)
	at com.intellij.psi.stubs.StubIndexImpl.processElements(StubIndexImpl.java:282)
	at com.intellij.psi.stubs.StubIndex.processElements(StubIndex.java:65)
	at org.angularjs.index.AngularIndexUtil.multiResolve(AngularIndexUtil.java:85)
	at org.angularjs.index.AngularIndexUtil.resolve(AngularIndexUtil.java:68)
	at org.angularjs.index.AngularIndexUtil.lambda$getAngularJSVersion$2(AngularIndexUtil.java:159)
	at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:54)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$1(CachedValueBase.java:240)
	at com.intellij.openapi.util.RecursionManager$1.doPreventingRecursion(RecursionManager.java:113)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:71)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:241)
	at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:43)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValueFromExHolder(CachedValuesManagerImpl.java:72)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:45)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:119)
	at org.angularjs.index.AngularIndexUtil.getAngularJSVersion(AngularIndexUtil.java:156)
	at org.angularjs.index.AngularIndexUtil.hasAngularJS(AngularIndexUtil.java:150)
	at org.angularjs.codeInsight.AngularJSHtmlExtension.isAvailable(AngularJSHtmlExtension.java:28)
	at com.intellij.xml.XmlExtension.calcExtension(XmlExtension.java:51)
	at com.intellij.xml.XmlExtension.lambda$getExtension$0(XmlExtension.java:34)
	at com.intellij.psi.util.CachedValuesManager.lambda$getCachedValue$0(CachedValuesManager.java:148)
	at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:54)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$1(CachedValueBase.java:240)
	at com.intellij.openapi.util.RecursionManager$1.doPreventingRecursion(RecursionManager.java:113)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:71)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:241)
	at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:43)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValueFromExHolder(CachedValuesManagerImpl.java:72)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:45)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:147)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:129)
	at com.intellij.xml.XmlExtension.getExtension(XmlExtension.java:34)
	at com.intellij.xml.XmlExtension.getExtensionByElement(XmlExtension.java:61)
	at com.intellij.psi.impl.source.xml.XmlTagImpl.computeNamespaceMap(XmlTagImpl.java:897)
	at com.intellij.psi.impl.source.xml.XmlTagImpl.lambda$getNamespaceMap$12(XmlTagImpl.java:867)
	at com.intellij.psi.util.CachedValuesManager.lambda$getCachedValue$0(CachedValuesManager.java:148)
	at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:54)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$1(CachedValueBase.java:240)
	at com.intellij.openapi.util.RecursionManager$1.doPreventingRecursion(RecursionManager.java:113)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:71)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:241)
	at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:43)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValueFromExHolder(CachedValuesManagerImpl.java:72)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:45)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:147)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:129)
	at com.intellij.psi.impl.source.xml.XmlTagImpl.getNamespaceMap(XmlTagImpl.java:866)
	at com.intellij.psi.impl.source.xml.XmlTagImpl.getNamespaceByPrefix(XmlTagImpl.java:785)
	at com.intellij.psi.impl.source.xml.XmlTagImpl.getNamespaceByPrefix(XmlTagImpl.java:791)
	at com.intellij.psi.impl.source.xml.XmlTagImpl.getNamespaceByPrefix(XmlTagImpl.java:791)
	at com.intellij.psi.impl.source.xml.XmlTagImpl.getNamespaceByPrefix(XmlTagImpl.java:791)
	at com.intellij.psi.impl.source.xml.XmlTagImpl.lambda$getNamespace$10(XmlTagImpl.java:768)
	at com.intellij.psi.util.CachedValuesManager.lambda$getCachedValue$0(CachedValuesManager.java:148)
	at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:54)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$1(CachedValueBase.java:240)
	at com.intellij.openapi.util.RecursionManager$1.doPreventingRecursion(RecursionManager.java:113)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:71)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:241)
	at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:43)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValueFromExHolder(CachedValuesManagerImpl.java:72)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:45)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:147)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:129)
	at com.intellij.psi.impl.source.xml.XmlTagImpl.getNamespace(XmlTagImpl.java:767)
	at com.intellij.psi.impl.source.html.HtmlTagImpl.getNamespace(HtmlTagImpl.java:98)
	at org.intellij.lang.xpath.xslt.XsltSupport.isXsltTag(XsltSupport.java:110)
	at org.intellij.lang.xpath.xslt.XsltSupport.isXsltAttribute(XsltSupport.java:105)
	at org.intellij.lang.xpath.xslt.XsltSupport.isXPathAttribute(XsltSupport.java:122)
	at org.intellij.lang.xpath.xslt.impl.XPathLanguageInjector.getLanguagesToInject(XPathLanguageInjector.java:140)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl.processInPlaceInjectorsFor(InjectedLanguageManagerImpl.java:421)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.probeElementsUp(InjectedLanguageUtil.java:348)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.enumerate(InjectedLanguageUtil.java:173)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.findInside(InjectedLanguageUtil.java:490)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.tryOffset(InjectedLanguageUtil.java:469)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.findElementAtNoCommit(InjectedLanguageUtil.java:316)
	at com.intellij.database.console.JdbcConsoleProvider.elementAt(JdbcConsoleProvider.java:313)
	at com.intellij.database.console.JdbcConsoleProvider.isSqlAtPlace(JdbcConsoleProvider.java:294)
	at com.intellij.database.actions.QueryActionBase.update(QueryActionBase.java:49)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:176)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performFastUpdate(ActionUtil.java:129)
	at com.intellij.ui.mac.touchbar.TBItemAnActionButton.updateAnAction(TBItemAnActionButton.java:89)
	at com.intellij.ui.mac.touchbar.TouchBar.lambda$updateActionItems$6(TouchBar.java:250)
	at com.intellij.ui.mac.touchbar.ItemsContainer.lambda$forEachDeep$2(ItemsContainer.java:144)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at com.intellij.ui.mac.touchbar.ItemsContainer.forEachDeep(ItemsContainer.java:139)
	at com.intellij.ui.mac.touchbar.TouchBar.forEachDeep(TouchBar.java:236)
	at com.intellij.ui.mac.touchbar.TouchBar.updateActionItems(TouchBar.java:245)
	at com.intellij.ui.mac.touchbar.TouchBar$UpdateTimerWrapper$1.run(TouchBar.java:300)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.runListenerAction(ActionManagerImpl.java:1492)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.notifyListeners(ActionManagerImpl.java:1481)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.actionPerformed(ActionManagerImpl.java:1465)
	at java.desktop/javax.swing.Timer.fireActionPerformed(Timer.java:317)
	at java.desktop/javax.swing.Timer$DoPostEvent.run(Timer.java:249)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:878)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:827)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:466)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:465)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```
## Error context
```
Last Action:       EditorEscape
Plugin Version:    0.11.1
OS Name:           Mac OS X
Java Version:      11.0.4
Java VM Vendor:    JetBrains s.r.o
Product Name:      IDEA
Full Product Name: IntelliJ IDEA
App Version Name:  IntelliJ IDEA
Is EAP:            false
App Build:         IU-192.7142.36
App Version:       2019.2.4
```

**Comment by aem-intellij-plugin-error-reporter** [comment]
## Description
*No description*
## Error context
```
Last Action:       
Plugin Version:    0.8.5
OS Name:           Linux
Java Version:      1.8.0_201
Java VM Vendor:    Oracle Corporation
Product Name:      IDEA
Full Product Name: IntelliJ IDEA
App Version Name:  IntelliJ IDEA
Is EAP:            false
App Build:         IU-182.4323.46
App Version:       2018.2.3
```

**Comment by karollewandowski** [comment]
Fixed in 0.11.2 - it will be released soon.
